### DISCUSSION TITLE
Github student developer pack is not open properly

### DISCUSSION THREAD

**Discussion by pradip70089** [community/community#40748] [Actions]
### Select Topic Area

Bug

### Body

Github student developer pack is not open properly.

-  First i click **[Github student developer pack](https://education.github.com/pack)** , 

![Screenshot (2)](https://user-images.githubusercontent.com/119783916/205483749-e0cd9179-2dc5-4429-91f2-aa581ea00ae6.png)

- Then see **522 gateway**  , why any problem.

![Screenshot (3)](https://user-images.githubusercontent.com/119783916/205483759-4c94ed8e-fd0f-4975-9ab2-36c72f5617c7.png)

**Comment by DSDmark** [comment]
Hi @pradip70089,

try to switch your network. and disable the firewall. I don't about Windows but some it firewall course some network issues. try maybe it helps.

**Reply by airtower-luna** [reply]
Please don't suggest disabling a firewall randomly. Sure, a misconfigured firewall can cause problems, but there's no indication that's the case here, so it's only encouraging unsafe behavior.

**Comment by airtower-luna** [comment]
That's an error message from the webserver, so it looks like something is broken on the GitHub end. If it persists, you might want to [submit a bug report](https://support.github.com/contact/bug-report) to GitHub support.

**Comment by karnik1110** [comment]
have you found the solution?
I am facing the same problem
